### PR TITLE
Add readmes for each package

### LINKED ISSUE #153: Create a brief README.md for each package to improve npm page
Add a brief README.md on each package under @fuels-ts.

- [ ] Decription, what it does
- [ ] Link to docks
- [ ] Link to umbrella package `fuels`
- [ ] License

### PR DESCRIPTION
**Pull request by camsjams**
Fixes #153

I've taken a first pass on each description, we can opine on the `description` of each package and the overall contents of the READMEs. **Each README here is identical aside from the description.**

I can make changes easily because they are generated using a base markdown file that I've created locally, which I've copied below.

<details><summary>See base markdown template</summary>

# `_PACKAGE_NAME_`

**_PACKAGE_NAME_** is a sub-module for interacting with **Fuel**.

_PACKAGE_DESC_

# Table of contents

- [Documentation](#documentation)
- [Usage](#usage)
  - [Installation](#installation)
  - [Full SDK Installation](#full-sdk-installation)
- [Contributing](#contributing)
- [Changelog](#changelog)
- [License](#license)

## Documentation

See [Fuel TypeScript SDK Documentation](https://fuellabs.github.io/fuels-ts/)

### Installation

```sh
yarn add _PACKAGE_NAME_
# or
npm add _PACKAGE_NAME_
```

### Full SDK Installation

Alternatively, we recommend you install the [complete SDK](https://github.com/FuelLabs/fuels-ts) using the umbrella package:

```sh
yarn add fuels
# or
npm add fuels
```

## Contributing

In order to contribute to `_PACKAGE_NAME_`, please see the main [fuels-ts](https://github.com/FuelLabs/fuels-ts) monorepo.

## Changelog

The `_PACKAGE_NAME_` changelog can be found at [CHANGELOG](./CHANGELOG.md).

## License

The primary license for `_PACKAGE_NAME_` is `Apache 2.0`, see [LICENSE](./LICENSE).

</details>

### CODE REVIEW

File: packages/abi-coder/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+```sh
+yarn add @fuel-ts/abi-coder
+# or
+npm add @fuel-ts/abi-coder
+```
+
+### Full SDK Installation
+
+Alternatively, we recommend you install the [complete SDK](https://github.com/FuelLabs/fuels-ts) using the umbrella package:
```
Comment (luizstacio) [inline packages/abi-coder/README.md:33]: ❤️

**Review by luizstacio — CHANGES_REQUESTED** [review]

File: packages/abi-coder/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+- [Usage](#usage)
+  - [Installation](#installation)
+  - [Full SDK Installation](#full-sdk-installation)
+- [Contributing](#contributing)
+- [Changelog](#changelog)
+- [License](#license)
+
+## Documentation
+
+See [Fuel TypeScript SDK Documentation](https://fuellabs.github.io/fuels-ts/)
```
[the remark this answers is not in the document]
Comment (camsjams) [reply packages/abi-coder/README.md:19]: Yes this is possible - good point!

[the remark this answers is not in the document]
Comment (camsjams) [reply packages/abi-coder/README.md]: Updated :+1:

File: typedoc.json [hunk 3]
```diff
@@ -3,15 +3,26 @@
...
-    "packages/contract",
-    "packages/abi-coder"
+    "packages/signer",
+    "packages/sparsemerkle'",
+    "packages/testcases",
+    "packages/transaction",
+    "packages/typechain-target-fuels",
+    "packages/wallet",
+    "packages/wallet-manager",
+    "packages/wordlists"
```
Comment (camsjams) [inline typedoc.json:25]: These are all packages aside from `fuels` and `sample-contract`

**Review by luizstacio — APPROVED** [review]
LGTM! Nice work

**Pull request merged**